.gulp/gulpfile.iced

```diff
@@ -24,7 +24,6 @@ task 'init', "" ,(done)->
 task 'test', "", ['regenerate'], (done) ->
   process.env.GOPATH = "#{basefolder}/test"
   await execute "glide install",          { cwd: './test/src/tests' }, defer code, stderr, stdout
-  await execute "go fmt ./generated/...", { cwd: './test/src/tests' }, defer code, stderr, stdout

```

Comment:
Why was this pushed to regeneration?  I'm not pushing back just curious.

Reply:
Because when generated code changes, the changes should be checked in. It should be already formatted code, but right now only generating does not format. I dont want to run tests just to format code.